File: MODUL 4/SOAL 4/PRAK304-2210817210026-Zulfa Auliya Akbar.c
#include <stdio.h>

int main(){

    int a;

    scanf("%d",&a);

    if(a == 0){
        printf("nol");
    }
    else if(a>0 && a<=9){
        printf("Satuan");
    }
    else if(a>=10 && a<=19){
        printf("Belasan");
    }
    else if(a>=20 && a<100){
        printf("Puluhan");
    }
    else{
        printf("Anda Menginput Melebihi Limit Bilangan");
    }

return 0;
}

File: MODUL 4/SOAL 4/PRAK304-2210817210026-Zulfa Auliya Akbar.py
a = int(input())

if a>=0:
    if a<100:
        if a==0:
            print("Nol")
        elif a>0:
            if a<10:
                print("Satuan")
            elif a<20:
                if a>=10:
                    print("Belasan")
            elif a>=20:
                print("Puluhan")
    else:
        print("Anda Menginput Melebihi Limit Bilangan")
else:
    print("Anda Menginput Melebihi Limit Bilangan")
        